back/app/models/idea.rb

```diff
@@ -234,7 +234,7 @@ def validate_creation_phase
       )
     end
 
-    if %w[ideation budgeting].include?(creation_phase.participation_method)
+    if creation_phase.transitive?
```

Comment:
A phase should not decide whether it is an error to set it as creation_phase in an input. The participation method object should do that. That is why we have a method `form_in_phase?`. So I think the conditional should be `unless participation_method_on_creation.form_in_phase?`, shouldn't it?